src/FileFormat/PdfStatistics.php

```diff
@@ -0,0 +1,217 @@
+<?php
+namespace CubeTools\CubeCommonBundle\FileFormat;
+
+/**
+ * Class handling statistics for PDF files.
+ */
+class PdfStatistics
+{
+    /**
+     * Because 72 pixels make one inch (~254 mm), this parameter shows, how big in mm is one pixel (for 72 ppi).
+     */
+    const RATIO_PIXEL_TO_MM = 0.352941176;
+
+    /**
+     * Suffix for rotated pages (can be empty).
+     */
+    const PAGE_ROTATED_SUFFIX = '';
+
+    /**
+     * Size of 4A0 format in 72 ppi
```

Comment:
never heard about the format 4A0. This probably means 4x a "A0". But this term seems to be used generally (found some search results). So I am fine with this.

Reply:
I found those formats here: http://www.papersizes.org/a-paper-sizes.htm . Hoped that I will fit into some of our papers. However, we can add here additional formats like those from B and C (envelope) series.